.github/workflows/docker_management.prune.yml

```diff
@@ -0,0 +1,27 @@
+name: Prune temporary docker images
+
+on:    
+  schedule:
+    - cron:  '15 4 * * *'
+
+  workflow_dispatch:
+
+jobs:
+  prune-images:
+    if: github.repository_owner == 'saheerb'
```

Comment:
Not sure about this check

Reply:
Yea, should have removed it. Will do in the next version